cdap-app-templates/cdap-etl/cdap-etl-api/src/main/java/co/cask/cdap/templates/etl/api/StageSpecification.java

```diff
@@ -29,6 +29,10 @@
   private final String description;
   private final List<Property> properties;
 
+  public StageSpecification(String name, String description, List<Property> properties) {
```

Comment:
We shouldn't need this. The one who new this spec (which is the configurer) should already know about the class.